models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,140 @@ class pp_cond_exp_mc_urbanczik_parameters
 
-/** @BeginDocumentation
-@ingroup Neurons
-@ingroup pp
-@ingroup cond
-
+Short description
++++++++++++++++++
 
-Name: pp_cond_exp_mc_urbanczik - Two-compartment point process neuron with conductance-based synapses
+Two-compartment point process neuron with conductance-based synapses
 
-Description:
+Description
++++++++++++
 
 pp_cond_exp_mc_urbanczik is an implementation of a two-compartment spiking
 point process neuron with conductance-based synapses as it is used
-in [1]. It is capable of connecting to a Urbanczik synapse.
+in [1]_. It is capable of connecting to an :doc:`Urbanczik synapse
+<urbanczik_synapse>`.
 
-The model has two compartments: soma and dendrite, labeled as s and p, respectively.
-There is one excitatory and one inhibitory conductance-based synapse onto each compartment.
-Each compartment can also receive current input from a current generator, and an external
-(rheobase) current can be set for each compartment.
+The model has two compartments: soma and dendrite, labeled as s and p,
+respectively. There is one excitatory and one inhibitory conductance-based
+synapse onto each compartment. Each compartment can also receive current input
+from a current generator, and an external (rheobase) current can be set for
+each compartment.
 
 Synapses, including those for injection external currents, are addressed through
 the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
-
-Parameters:
+.. _multicompartment-models:
+
+Multicompartment models and synaptic delays
++++++++++++++++++++++++++++++++++++++++++++
+
+Note that in case of multicompartment models which represent the dendrite
+explicitly, the interpretation of the synaptic delay in NEST requires a careful
+consideration. In NEST, the delay is at least one simulation time step and is
+assumed to be located entirely at the postsynaptic side. For point neurons, it
+represents the time it takes for an incoming spike to travel along the
+postsynaptic dendrite before it reaches the soma, see :ref:`panel a)
+<fig-multicompartment>`. Conversely, if the synaptic weight depends on the
+state of the postsynaptic neuron, the delay also represents the time the
+information on the state propagates back through the dendrite to the synapse.
+
+For multicompartment models, this amounts to positioning the delay directly
+behind the incoming synapse, i.e., before the first dendritic compartement on
+the postsynaptic side, see :ref:`panel b) <fig-multicompartment>`. Therefore,
+the delay specified in the synapse model does *not* account for any delay that
+might be associated with information traveling through the explicitly modeled
+dendritic compartments.
+
+In case of the Urbanczik synapse, the change of the synaptic weight is driven by
+an error signal which is the difference between the firing rate of the soma
```

Comment:
```suggestion
an error signal, which is the difference between the firing rate of the soma
```
What do you mean by error signal?

Reply:
@jessica-mitchell I tried to include the definition of "error signal" in the next half sentence "... error signal which is the difference between the firing rate ..." Apparently, this was not clearly expressed. Would it help to write "... error signal which is defined as the difference between the firing rate ..."? One could also replace the term "error signal" by "prediction error" which is used in [1]. However, as far as I know "error signal" is also quite common in the context of synaptic plasticity.

Reply:
If error signal is common terminology, then I'm good with it. It was just something I wasn't familiar with.

Reply:
Okay, then let's keep "error signal".